### PR TITLE
chore(generator): lock jsonnet library versions

### PR DESCRIPTION
**Pull request by Duologic**
A bit of housekeeping:
- Lock versions to ensure reproducible builds 
  (replaces #125)
- Add make targets for all current versions

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
+	./scripts/generate.sh ${LATEST} && \
+	./scripts/generate_latest.sh ${LATEST}
+
+regenerate: gen/grafonnet-v*
+
+# is latest, use `make generate` instead
+#gen/grafonnet-v10.0.0:
+#	./scripts/generate.sh v10.0.0
+
+gen/grafonnet-v9.5.0:
```
Comment (julienduchesne) [inline Makefile:17]: Why 9.5.0 and not the latest patch? Same for all versions

File: Makefile [hunk 2]
```diff
@@ -1,10 +1,24 @@
-.PHONY: generate localmkdocs test
+.PHONY: generate regenerate localmkdocs test
 
 LATEST := v10.0.0
```
Comment (julienduchesne) [inline Makefile:3]: Latest would be 10.1, no?

File: Makefile (hunk 1 above)
Comment (Duologic) [reply Makefile:17]: This builds on the grok pipeline, following the major releases: https://github.com/grafana/grok/tree/main/jsonnet

File: Makefile (hunk 2 above)
Comment (Duologic) [reply Makefile:3]: Working on that one here: https://github.com/grafana/grok/pull/34
Will do that in a separate PR.

**Review by julienduchesne — APPROVED** [review]

**Pull request merged**